### DISCUSSION TITLE
Is there a way to make the Primereact DataTable horizontal scrollbar float?

### DISCUSSION THREAD

**Discussion by Amit-ga** [primefaces/community#799] [PrimeReact] [answered]
From my research so far, ive reached the conclusion that the table's horizontal scrollbar is placed at the bottom of the table and there is no way to change that. 
My table is pretty big and presents ~700 lines per page (I do not want to show less lines per page).
I also have many columns which do not all fit into the screen, so the user has to scroll all the way down just to get to the horizontal scrollbar and scroll sideways and then go back up to where he was. that is, of course, very uncomfortable.
Any way wo make the table horizontal scrollbar float and appear on the bottom of the screen at all times regardless of the size of the table itself?
Thanks.

**Comment by melloware** [comment, marked as answer]
It is most likely placed there by the browser CSS Overflow property: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow  So its not PrimeReact placing the scrollbar on the right and bottom it is the browser doing it to the container based on CSS overflow.

**Reply by Amit-ga** [reply]
Thank you for you answer.
It seems to me that the vertical scrollbar is placed in the right side of the page by the browser, but i dont think that is the case for the horizontal scrollbar. It seems to be a part of the table itself.
I can see the vertical scrollbar at all times regardless of the size of the table, which is the behavior I want to apply to the horizontal scrollbar.
Here is a picture for example:
![image](https://github.com/primefaces/community/assets/115641126/7a2910be-8941-4ace-96d9-0f586e9faaf6)
As you can see, the vertical scroll is just fine, but the horizontal scrollbar appears to be connected to the table component. I want to "pull it out" of the table, and place it as part of the browser itself. Is that possible?
Thank you.

**Reply by melloware** [reply]
You will have to review the HTML and CSS being generated but I don't think its possible.   I have a feeling its not being "generated" by the component its that internal `<DIV>` for the inner table has `overflow on it.

**Reply by Amit-ga** [reply]
thank you. I am rather new to this, any way to change a component's inner div?

**Reply by melloware** [reply]
Probably not without a code change but you can study the code and see.
